fvm/errors/codes.go

```diff
@@ -47,6 +47,7 @@ const (
 	ErrCodeInvalidEnvelopeSignatureError ErrorCode = 1009
 
 	// base errors 1050 - 1100
+	// Deprecated: No longer used.
 	ErrCodeFVMInternalError            ErrorCode = 1050
```

Comment:
why still keeping this error?

Reply:
i'm just following janez's convention

Reply:
my guess is we want to keep these around just to be sure we don't accidentally reuse the same number